Utilities/File.cpp

```diff
@@ -322,12 +322,53 @@ std::shared_ptr<fs::device_base> fs::set_virtual_device(const std::string& name,
 
 std::string fs::get_parent_dir(const std::string& path)
 {
-	// Get (basically) processed path
-	const auto real_path = fs::escape_path(path);
```

Comment:
maybe fix escape_path to work with network locations.

Reply:
Parent dir always worked scanning from the end, and it was OK for mimimal purposes. The problem with renaming arises not so often and there are different approaches to it and other problems (such as case sensitivity). I have one fundamental in mind which I mentioned as "journaled VFS".